src/components/ClauseComponent.js

```diff
@@ -1,33 +1,76 @@
+/* React */
 import React from 'react';
 import PropTypes from 'prop-types';
-import { Header, Segment } from 'semantic-ui-react';
+import { Segment } from 'semantic-ui-react';
+
+/* Styling */
+import * as styles from './styles';
+
+/* Icons */
+import * as deleteIcon from '../icons/trash';
+// import * as addIcon from '../../assets/icons/add';
```

Comment:
don't need line 11?

Reply:
Ah yes, this was to set up work for the navigation component which will use the `add` icon. I'll remove that for this PR.